application/src/main/java/org/togetherjava/tjbot/commands/reminder/ReminderCommand.java

```diff
@@ -193,28 +199,21 @@ private MessageCreateData createListMessage(Result<PendingRemindersRecord> pendi
     }
 
     private List<Button> createPageTurnButtons(int pageNumber, int totalPages) {
-        int previousButtonTurnPageBy = -1;
-        Button previousButton =
-                createPageTurnButton(PREVIOUS_BUTTON_LABEL, pageNumber, previousButtonTurnPageBy);
+        Button previousButton = Button.primary(generateComponentId(String.valueOf(pageNumber)),
+                PREVIOUS_BUTTON_LABEL);
         if (pageNumber <= 1) {
             previousButton = previousButton.asDisabled();
         }
 
-        int nextButtonTurnPageBy = 1;
         Button nextButton =
-                createPageTurnButton(NEXT_BUTTON_LABEL, pageNumber, nextButtonTurnPageBy);
+                Button.primary(generateComponentId(String.valueOf(pageNumber)), NEXT_BUTTON_LABEL);
```

Comment:
No need to call String.valueOf twice. You should just store the String representation in a variable like `pageNumberString`. This would also make it shorter and your button creation would probably fit in one line.

Reply:
last time i did pagination (in audit command) zabu told me to do it this way, so gonna keep it like this

Reply:
I am pretty sure he did not tell you to duplicate code

Reply:
ohh yea he told me to make a separate method but no method now so ig i can make a variable